### PR TITLE
[T3A3][W10-A4]Inaba Kazuhiro

### PR DESCRIPTION
**Pull request by kazuhiroinaba**

### CODE REVIEW

**Comment by inaba1231** [comment]
ready for review

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -25,6 +25,11 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         this.feedbackToUser = feedbackToUser;
         this.relevantPersons = relevantPersons;
     }
+    
+    /** Returns feedback message to be shown to the user. */
+    public String getFeedbackToUser() {
+    	return feedbackToUser;
+    }
```
Comment (damithc) [inline src/seedu/addressbook/commands/CommandResult.java]: Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.

File: src/seedu/addressbook/data/person/Address.java [hunk 2]
```diff
@@ -12,7 +12,8 @@
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
 
-    public final String value;
+    // public final String value;
```
Comment (damithc) [inline src/seedu/addressbook/data/person/Address.java]: Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.

File: src/seedu/addressbook/data/person/AddressPart.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
+public class AddressPart {
```
Comment (damithc) [inline src/seedu/addressbook/data/person/AddressPart.java]: Missing header comment. All non trivial classes/methods should have javadoc format header comments.

**Comment by damithc** [comment]
Good effort @inaba1231 
Some comments added. Please close the PR after reading comments.

**Comment by damithc** [comment]
@inaba1231 this a reminder to close this PR after reading comments. If PRs remain open, we will assume you are not interested in reading our comments and therefore we will stop reviewing your PRs.

**Pull request closed without merging**